docs/src/chapters/prezentacja.adoc

```diff
@@ -1 +1,287 @@
+:mobile-size: pdfwidth=30%
+
 == Prezentacja w działaniu
+
+W procesie implementacji projektu wytworzono trzy główne komponenty, czyli serwer, aplikację mobilną oraz aplikację
+internetową. API serwera nie zostało zaprojektowane do użytku publicznego. Jego jedynym zadaniem jest obsługiwanie
+_backend'owej_ części projektu. W tym rozdziale zostanie zaprezentowana aplikacja mobilna oraz aplikacja webowa.
+Pokazane zostaną wszystkie dostępne funkcjonalności i procesy.
+
+Przy wejściu do aplikacji internetowej użytkownik trafia na stronę główną. Zawiera ona krótki komunikat powitalny
```

Comment:
zamiast trafia, można dać zostaje przekierowany, chyba lepiej to brzmi, ale to już jak uważasz

Reply:
Z a przekierowanie rozumiem z miejsca do miejsca. Tutaj jest coś, co jest or razu, więc przekierowanie nie pasuje. Dla mnie "trafia" tutaj jest git.